src/backend/cuda/device_manager.cpp

```diff
@@ -52,7 +52,7 @@ using std::stringstream;
 
 namespace cuda {
 
-void findJitDevCompute(pair<int, int>& prop) {
+void checkAndSetDevMaxCompute(pair<int, int>& prop) {
```

Comment:
I changed this to reflect what the function is doing in a better way. It was misleading earlier.